app/javascript/listings/components/SelectedTags.jsx

```diff
@@ -0,0 +1,39 @@
+import { h } from 'preact';
+import { selectedTagsPropTypes } from '../../src/components/common-prop-types';
+
+const SelectedTags = ({ tags, onClick, onKeyPress }) => {
+  return (
+    <section>
+      {tags.map((tag) => (
+        <span
+          className="classified-tag"
+          key={tag.id}
+          id={`selected-tag-${tag.id}`}
+        >
+          <a
+            href="/listings?tags="
+            className="tag-name"
+            onClick={onClick}
+            data-no-instant
+          >
+            <span>{tag}</span>
+            <span
```

Comment:
Since this is acting as a button, simply use a `<button />`. Then there is no need for the `role` or `tabIndex` props.

Reply:
Good suggestion. :+1: